src/main/java/ebay/carina/components/ios/FilterIOS.java

```diff
@@ -0,0 +1,51 @@
+package ebay.carina.components.ios;
+
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import ebay.carina.components.common.FilterBase;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.math.BigDecimal;
+
+public class FilterIOS extends FilterBase {
+    @FindBy(xpath = "//span[@class = 'srp-multi-aspect__flyout__btn-label' and text() = '%s']")
+    private ExtendedWebElement filterOption;
+
+    @FindBy(id = "s0-17-23-6-3-4[0]-3-1-13-9[1]-0-0-2-flyout-7-5-4-9-10-textbox")
```

Comment:
Locators here looks unstable, find option to locate element without autogenerated attributes